junit-runtime/src/main/scala/scala/scalanative/junit/Bootstrapper.scala

```diff
@@ -36,6 +36,7 @@ trait Bootstrapper {
  */
 final class TestMetadata(
     val name: String,
-    val ignored: Boolean,
+    val testClassIgnored: Boolean,
```

Comment:
IIUC, this will either be `true` for all tests in a class, or `false` for all tests in a class. If that's the case, it doesn't make sense to record it in the per-test metadata. Instead, we should have a `final class TestClassMetadata`, and a new `def classMetadata(): TestClassMetadata` in `Bootstrapper`.